release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3326,6 +3326,62 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.34 --pullspecs
+----
+
+[id="ocp-4-14-34-enhancements_{context}"]
+==== Enhancements
+
+[id="ocp-4-14-34-timeout-ingress_{context}"]
+===== Adding connectTimeout tuning option to Ingress Controller API
+
+* The IngressController API is updated with a new tuning option, `ingresscontroller.spec.tuningOptions.connectTimeout`, which defines how long the router will wait for a response when establishing a connection to a backend server. (link:https://issues.redhat.com/browse/OCPBUGS-36555[*OCPBUGS-36555*])
+
+[id="ocp-4-14-34-insights-operator-collecting-prometheus-alertmanager_{context}"]
+===== Collecting Prometheus and AlertManager resources with the Insights Operator
+
+* The Insights Operator (IO) now collects the `Prometheus` and `AlertManager` resources in addition to the `openshift-monitoring` custom resource. (link:https://issues.redhat.com/browse/OCPBUGS-36380[*OCPBUGS-36380*])
+
+[id="ocp-4-14-34-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, AWS HyperShift clusters leveraged their VPC's primary CIDR range to generate security group rules on the data plane. As a result, installing AWS HyperShift clusters into an AWS VPC with multiple CIDR ranges could cause the generated security group rules to be insufficient. With this update, security group rules are generated based on the provided Machine CIDR range instead to resolve this issue. (link:https://issues.redhat.com/browse/OCPBUGS-36159[*OCPBUGS-36159*])
+
+* Previously, when a pod specified a node selector with no matching nodes, the kube-scheduler panicked with the following error: _Observed a panic: integer divide by zero_. With this release, the issue in the kube-scheduler code base is resolved, and the kube-scheduler no longer panics when a pod specifies a node selector with no matching nodes. (link:https://issues.redhat.com/browse/OCPBUGS-36397[*OCPBUGS-36397*])
+
+* Previously, installing an Operator could sometimes fail if the same Operator had been previously installed and uninstalled. This was due to a caching issue. With this release, {olm} is updated to install the Operator correctly in this scenario and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36452[*OCPBUGS-36452*])
+
+* Previously, the Ingress Operator could not successfully update the canary route because the Operator did not have permission to update `spec.host` or `spec.subdomain` on an existing route. With this release, the required permission is added to the cluster role for the Operator's ServiceAccount and the Ingress Operator can update the canary route. (link:https://issues.redhat.com/browse/OCPBUGS-36467[*OCPBUGS-36467*])
```

Comment:
Per our guidelines on API objects, you can lowercase when talking generically about things like service accounts: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

If you need to literally talk about a `ServiceAccount` object/resource, then you'd need to put it in backticks (and say "object" or "resource" after it.

```suggestion
* Previously, the Ingress Operator could not successfully update the canary route because the Operator did not have permission to update `spec.host` or `spec.subdomain` on an existing route. With this release, the required permission is added to the cluster role for the Operator's service account and the Ingress Operator can update the canary route. (link:https://issues.redhat.com/browse/OCPBUGS-36467[*OCPBUGS-36467*])
```